packages/nextjs/app/example-ui/page.tsx

```diff
@@ -0,0 +1,31 @@
+import type { Metadata, NextPage } from "next";
+import { ContractData } from "~~/components/example-ui/ContractData";
+import { ContractInteraction } from "~~/components/example-ui/ContractInteraction";
+
+export const metadata: Metadata = {
+  title: "Example UI",
+  description: "Example UI created with 🏗 Scaffold-ETH 2, showcasing some of its features.",
+  openGraph: {
+    title: "Example UI",
+    description: "Example UI created with 🏗 Scaffold-ETH 2, showcasing some of its features.",
+  },
+  twitter: {
+    title: "Example UI",
+    description: "Example UI created with 🏗 Scaffold-ETH 2, showcasing some of its features.",
+  },
+};
+
+const ExampleUI: NextPage = () => {
+  return (
+    <>
+      <link rel="preconnect" href="https://fonts.googleapis.com" />
```

Comment:
better to use [official way](https://nextjs.org/docs/app/building-your-application/optimizing/fonts#google-fonts)